dev/django.Dockerfile

```diff
@@ -1,13 +1,31 @@
 FROM python:3.11-slim
+
+# Set some behaviors for Python.
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
 # Install system librarires for Python packages:
 # * psycopg2
 RUN apt-get update && \
     apt-get install --no-install-recommends --yes \
     libpq-dev gcc libc6-dev git && \
     rm -rf /var/lib/apt/lists/*
 
-ENV PYTHONDONTWRITEBYTECODE 1
-ENV PYTHONUNBUFFERED 1
+# Install pre-commit. Use `pip` instead of `apt` in order to get a recent
+# version instead of whatever is in the image's package lists.
+RUN pip install pre-commit
```

Comment:
Should we just add `pre-commit` to the `dev` extra in `setup.py`? That way, it will be installed in the docker image by the existing `pip install` step without needing a separate one for pre-commit. Also, it will be automatically installed for native users when they do a `pip install -e .[dev]`. We already include `tox` there, so it seems like adding `pre-commit` makes sense.

Reply:
Good idea. See 22c8fdf57a5264093b252760cd0ef518ea20c67f.